src/hotspot/share/gc/g1/g1YoungGCPostEvacuateTasks.cpp

```diff
@@ -465,6 +465,7 @@ class FreeCSetStats {
 
   void report(G1CollectedHeap* g1h, G1EvacInfo* evacuation_info) {
     evacuation_info->set_regions_freed(_regions_freed);
+    evacuation_info->set_collection_set_used_before(_after_used_bytes + _before_used_bytes);
```

Comment:
The ordering, `before + after`, is more natural, IMO.

Reply:
Fixed